src/main/java/org/apache/mesos/offer/ResourceUtils.java

```diff
@@ -203,6 +203,17 @@ public static Resource getExpectedRanges(
         return resBuilder.build();
     }
 
+    public static TaskInfo.Builder serializeCommandInfo(Protos.TaskInfo.Builder builder) {
+        Protos.CommandInfo command = builder.getCommand();
+
+        if (builder.hasExecutor() && builder.getExecutor().hasCommand()) {
```

Comment:
The second portion of this if clause i.e. `builder.getExecutor().hasCommand()` is not necessary.